api/projects/tests/test_serializers.py

```diff
@@ -0,0 +1,39 @@
+from projects.serializers import ProjectSerializer
+
+
+def test_ProjectSerializer_get_is_identity_migration_done_calls_dynamo_wrapper_with_correct_arguments(
+    mocker, project
+):
+    # Given
```

Comment:
Couldn't we simplify this to (using the trick I learned from you 😄): 

```python
mock_dynamo_wrapper = mocker.MagicMock()
mock_dynamo_wrapper.is_migration_done.return_value = True
mocker.patch("projects.serializers.DynamoIdentityWrapper", return_value=mock_dynamo_wrapper)
```

I think this feels slightly neater?

Reply:
Yep, done